minecraft_servers/abbelsmp/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+    "server_name": "abbelsmp",
+    "nice_name": "AbbelSMP",
+    "direct_ip": "play.abbelsmp.de",
+    "server_wildcards": [
+      "%.abbelsmp.de"
+    ],
+    "supported_languages": [
+       "de"
+    ],
+    "social": {
+      "twitter": "vinnipapi",

```

Comment:
Links to private twitter account; either server account or remove

Reply:
Removed